planner/core/planbuilder.go

```diff
@@ -374,7 +374,12 @@ func NewPlanBuilder(sctx sessionctx.Context, is infoschema.InfoSchema, processor
 
 // Build builds the ast node to a Plan.
 func (b *PlanBuilder) Build(ctx context.Context, node ast.Node) (Plan, error) {
-	b.optFlag = flagPrunColumns
+	b.optFlag = flagPrunColumns // | flagPrunColumnsAgain
```

Comment:
Please don't use comment prefix to delete it. 😂

Reply:
Sorry, fixed.